homeassistant/components/rako/bridge.py

```diff
@@ -0,0 +1,124 @@
+"""Module representing a Rako Bridge."""
+from __future__ import annotations
+
+import asyncio
+from asyncio import Task
+import logging
+
+from python_rako.bridge import Bridge
+from python_rako.helpers import convert_to_brightness, get_dg_listener
+from python_rako.model import ChannelStatusMessage, SceneStatusMessage, StatusMessage
+
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .light import RakoLight
+from .model import RakoDomainEntryData
+from .util import create_unique_id
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RakoBridge(Bridge):
+    """Represents a Rako Bridge."""
+
+    def __init__(
+        self,
+        host: str,
+        port: int,
+        name: str,
+        mac: str,
+        entry_id: str,
+        hass: HomeAssistant,
+    ) -> None:
+        """Init subclass of python_rako Bridge."""
+        super().__init__(host, port, name, mac)
+        self.entry_id = entry_id
+        self.hass = hass
+
+    @property
+    def _light_map(self) -> dict[str, RakoLight]:
+        rako_domain_entry_data: RakoDomainEntryData = self.hass.data[DOMAIN][self.mac]
```

Comment:
This does not look like the right variable. My impression is that `hass.data[DOMAIN]` has another key with the `RakoDomainEntryData` set.

But can RakoDomainEntryData be an input into the constructor instead? I don't; think a hass data lookup is really needed given `async_setup_entry` has this.